src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -122,7 +152,7 @@ class JSONSerializer(Serializer):
         callable with JSONSerializer.
 
     Args:
-        schema_str (str): `JSON Schema definition. <https://json-schema.org/understanding-json-schema/reference/generic.html>`_
+        schema_str (str, Schema): `JSON Schema definition. <https://json-schema.org/understanding-json-schema/reference/generic.html>`_ as either a string or a `Schema` instance.  Note that string definitions cannot reference other schemas. For referencing other schemas, use a Schema instance.
```

Comment:
Accepts schema in the form of String or Schema (link Schema class) instance.

Reply:
Updated.